src/formatter.js

```diff
@@ -39,6 +39,11 @@ class Formatter {
             item.topics = _.map(item.topics, Utils.formatHexStr);
 
             const methodAbi = _.find(metadataObj.abi, { name: eventName });
+            if (methodAbi === undefined) {
+              console.warn(`Error: Could not find method in ABI for ${eventName}`);
```

Comment:
Usually people don't write "methodAbi === undefined" in javascript cuz there could be a var called 'undefined'. Although that is extremely unlikely to be the case in this func I suggest we use _.isUndefined(methodAbi) to be consistent.

Reply:
good point. keep forgetting lodash does everything, fixed